examples/basic/android/app/build.gradle

```diff
@@ -108,6 +111,7 @@ android {
         release {
             signingConfig signingConfigs.debug
             minifyEnabled enableProguardInReleaseBuilds
+            matchingFallbacks = ['release', 'debug']
```

Comment:
why is this needed?

Reply:
i don't remember exactly why i add it. i will remove it in the next commit :D